flink-connector-obkv-hbase/src/test/java/com/oceanbase/connector/flink/OBKVHBaseConnectorITCase.java

```diff
@@ -43,6 +44,13 @@ public Map<String, String> getOptions() {
         return options;
     }
 
+    @AfterEach
+    public void tearDown() {
```

Comment:
Resources annotated with `@Rule` and `@ClassRule` should not be started or stopped manually.